Library/Formula/muparserx.rb

```diff
@@ -0,0 +1,44 @@
+class Muparserx < Formula
+  homepage "http://muparser.beltoforion.de/"
+
+  stable do
```

Comment:
These don't need to go inside a `stable` block unless a stable-only patch or resource accompanies them.